content/zh-cn/case-studies/daocloud/index.html

```diff
@@ -0,0 +1,263 @@
+heading_background: /images/case-studies/daocloud/banner1.jpg
+heading_title_logo: /images/daocloud-light.svg
+subheading: >
+  为数字世界寻找全局最优解
+case_study_details:
+  - Company:「DaoCloud 道客」
+  - Location: 中国上海
+  - Industry: 云原生
+---
+
+<!--
+title: DaoCloud Case Study
+linkTitle: DaoCloud
+case_study_styles: true
+cid: caseStudies
+logo: daocloud_featured_logo.svg
+
+css: /css/style_daocloud.css
+new_case_study_styles: true
+heading_background: /images/case-studies/daocloud/banner1.jpg
+heading_title_logo: /images/daocloud-light.svg
+subheading: >
+  Seek Global Optimal Solutions for Digital World
+case_study_details:
+  - Company: DaoCloud
+  - Location: Shanghai, China
+  - Industry: Cloud Native
+-->
+
+<!--
+<h2>Challenges</h2>
+-->
+<h2>挑战</h2>
+
+<!--
+<p><a href="https://www.daocloud.io/en/">DaoCloud</a>, founded in 2014, is an innovation leader in the field of cloud native. It boasts independent intellectual property rights of core technologies for crafting an open cloud platform to empower the digital transformation of enterprises.</p>
+-->
+<p>作为云原生领域的创新领导者，<a href="https://www.daocloud.io/en/">DaoCloud</a> 成立于 2014 年底，拥有自主知识产权的核心技术，致力于打造开放的云操作系统，为企业数字化转型赋能。</p>
+
+<!--
+<p>DaoCloud has been engaged in cloud native since its inception. As containerization is crucial for cloud native business, a cloud platform that does not have containers as infrastructure is unlikely to attract its potential users. Therefore, the first challenge confronting DaoCloud is how to efficiently manage and schedule numerous containers while maintaining stable connectivity between them.</p>
+-->
+<p>「DaoCloud 道客」从一开始就在做云原生。由于容器化对于云原生业务至关重要，没有容器作为基础设施的云平台不太可能吸引其潜在用户。因此，「DaoCloud 道客」面临的第一个挑战就是如何高效地管理和调度众多的容器，同时保持它们之间的稳定连接。</p>
+
+<!--
+<p>As cloud native technology gains momentum, cloud native solutions proliferate like mushrooms after rain. However, having more choices is not always a good thing, because choosing from various products to globally maximize benefits and minimize cost is always challenging and demanding. Therefore, another obstacle ahead of DaoCloud is how to pick out the best runner in each field and organize them into one platform that can achieve global optimum for cloud native.</p>
+-->
+<p>随着云原生技术的蓬勃发展，云原生解决方案如雨后春笋般激增。然而，拥有更多选择并不总是一件好事，因为从各种产品中进行选择以在全球范围内实现利益最大化和成本最小化总是具有挑战性和要求的。因此，摆在「DaoCloud 道客」面前的另一个障碍是如何在各个领域中挑选出最好的跑者，并将他们组织到一个平台上，实现云原生的全局最优。</p>
+
+<!--
+<h2>Solutions</h2>
+-->
+<h2>解决方案</h2>
+
+<!--
+<p>As the de facto standard for container orchestration, Kubernetes is undoubtedly the preferred container solution. Paco Xu, head of the Open Source and Advanced Development team at DaoCloud, stated, "Kubernetes is a fundamental tool in the current container ecosystem. Most services or applications are deployed and managed in Kubernetes clusters."</p>
+-->
+<p>Kubernetes 作为容器编排的事实标准，无疑是首选的容器解决方案。「DaoCloud 道客」架构师兼开源团队负责人徐俊杰（Paco）表示 “Kubernetes 是目前容器生态里面比较基础的一环，绝大多数服务都是基于 Kubernetes 部署的，应用绝大多数都是在 Kubernetes 集群中运行和管理。“</p>
```

Comment:
Not sure whether it's suitable to show the real name in this article.

Reply:
My fault, sorry

Reply:
Can you review again, thanks!